frontend/src/actions/users.js

```diff
@@ -164,6 +167,31 @@ export function fetchUserDetail(userId) {
   };
 }
 
+export function fetchMyTransactions() {
+  return dispatch => {
+    console.log('fetch here');
```

Comment:
Lets not commit personal log messages to master. If you want to include logs in the project, try using a more substantial log message.